client/src/views/HowItWorks.tsx

```diff
@@ -1,133 +1,98 @@
 import * as React from 'react';
 import Typography from '@mui/material/Typography';
 import { makeStyles } from '@mui/styles';
-import { Box, Grid, Container } from '@mui/material';
+import { Box, Grid } from '@mui/material';
 
 import type { Theme } from '@mui/material/styles';
+import HowItWorksCards from './HowItWorksCards';
+import HowItworks1 from '../assets/HowItWorks/HowItWorks1.svg';
+import HowItworks2 from '../assets/HowItWorks/HowItWorks2.svg';
+import HowItworks3 from '../assets/HowItWorks/HowItWorks3.svg';
+import HowItworks4 from '../assets/HowItWorks/HowItWorks4.svg';
 
-import QuestionList from '../components/QuestionList';
-import { BulletGrid, InstructionGrid } from '../components/DisplayGrids';
+const loremIpsum = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. ';
 
-const loremIpsum =
-  'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Facilisis placerat et, at vel tristique. Ac, gravida in quam gravida. Vel pretium nunc cursus donec enim. Sapien facilisis mauris justo, augue pharetra. Dignissim euismod fermentum sit gravida ut.';
-
-const nonprofitInstructionList = [
+const nonprofitinstructionsArray = [
   {
-    title: '1. Post your Need.',
-    body: loremIpsum,
-    image: 'https://picsum.photos/seed/nonprofit/600',
+    title: 'Sign up with Givingful',
+    body: 'Create a profile to let our community know about your nonprofit.',
+    image: HowItworks1,
```

Comment:
I think we could enhance readability by renaming all these images to what they do, for example: `sign-up-with-givingful.svg`